test_main_objs.py

```diff
@@ -6,6 +6,31 @@ def get_crim_model(file):
     root = "https://raw.githubusercontent.com/CRIM-Project/CRIM-online/master/crim/static/mei/MEI_3.0/"
     return CorpusBase([root + file]).scores[0]
 
+def test_get_semi_flat_parts_name():
+    """
+    Make sure that we could have correct names for each part.
+    Somewhat make sure that we could get all parts in the score.
+    """
+    hardcoded_names = FILES_PART_NAMES
+    for i in range(len(TEST_FILES)):
+        file = TEST_FILES[i]
+        model = get_crim_model(file)
+        names = model._getPartNames()
+        for j in range(len(hardcoded_names)):
+            assert(names[j] == hardcoded_names[i][j])
+
+def test_get_note_rests():
```

Comment:
I know it's pretty self-explanatory here, but a docstring would be great to make it super obvious for someone looking at the codebase for the first time. Could specify the class and method, like "Tests ImportedPiece.getNoteRest...".